src/Nav.js

```diff
@@ -154,6 +156,7 @@ class Nav extends React.Component {
         this.handleTabKeyDown.bind(this, onSelect || (()=>{})),
         onKeyDown
       );
+      tabIndex = isActive ? tabIndex : -1;
```

Comment:
This is the only part that really matters. Otherwise everything gets a weird `tabIndex`.